plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -166,4 +171,56 @@ public IcebergConfig setProjectionPushdownEnabled(boolean projectionPushdownEnab
         this.projectionPushdownEnabled = projectionPushdownEnabled;
         return this;
     }
+
+    @NotNull
+    public DataSize getSplitTargetSize()
+    {
+        return splitTargetSize;
+    }
+
+    @Config("iceberg.split-target-size")
+    public IcebergConfig setSplitTargetSize(DataSize splitTargetSize)
+    {
+        this.splitTargetSize = splitTargetSize;
+        return this;
+    }
+
+    @NotNull
+    public DataSize getSplitMetadataTargetSize()
+    {
+        return splitMetadataTargetSize;
+    }
+
+    @Config("iceberg.split-metadata-target-size")
+    public IcebergConfig setSplitMetadataTargetSize(DataSize splitMetadataTargetSize)
+    {
+        this.splitMetadataTargetSize = splitMetadataTargetSize;
+        return this;
+    }
+
+    public int getSplitPlanningLookback()
+    {
+        return splitPlanningLookback;
+    }
+
+    @Config("iceberg.split-planning-lookback")
+    @ConfigDescription("The number of bins considered when trying to pack the next file split into a scan task.")
+    public IcebergConfig setSplitPlanningLookback(int splitPlanningLookback)
+    {
+        this.splitPlanningLookback = splitPlanningLookback;
+        return this;
+    }
+
+    @NotNull
+    public DataSize getSplitOpenFileCost()
+    {
+        return splitOpenFileCost;
+    }
+
+    @Config("iceberg.split-open-file-cost")
+    public IcebergConfig setSplitOpenFileCost(DataSize splitOpenFileCost)
```

Comment:
As a user, when would i want to set it?

also, please add a ConfigDescription

Reply:
Got it. A